notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -1,1140 +1,1273 @@
 {
```

Comment:
<p>Move title to top of links cell.</p><p> </p><p>Split Overview ... Costs into 4 separate cells.</p><p> </p><p>Move Dataset after Objective.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>